userdocs/diagnostics/multiple-class-inheritance.md

```diff
@@ -0,0 +1,37 @@
+# Multiple Inheritance of Classes
```

Comment:
I'd suggest either just "Multiple Inheritance" or "Inheritance from Multiple Classes" if you want to elaborate, because "of" here is kind of ambiguous.